packages/babel-cli/README.md

```diff
@@ -9,7 +9,7 @@ There are some shell-executable utility scripts, `babel-external-helpers.js` and
 ## Install
 
 ```sh
-npm install --save-dev @babel/cli
+npm install --save-dev @babel/core @babel/cli
```

Comment:
I guess we should remove babel-core in the deps for babel-cli too

Reply:
Woops, saw that it had the peerDep and didn't realize it still had it in `dependencies`.